server/kotc_game/src/game/column.rs

```diff
@@ -32,8 +32,12 @@ impl Column {
         let number_of_cards = self.cards.len();
         if self.cards.len() >= 2 {
             let mut previous_card = self.cards.get_mut(number_of_cards-2).unwrap();
-            previous_card.revealed = true;
-            Some(previous_card.character)
+            if previous_card.revealed {
```

Comment:
Could you give me example of how can this happen? (prev card is already revealed)

Reply:
Killer